server/src/test/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorSpec.scala

```diff
@@ -27,7 +27,7 @@ import scala.concurrent.{Await, Promise}
 class WorkflowExecutionActorSpec extends CromwellTestKitSpec with AnyFlatSpecLike with Matchers with BeforeAndAfter with WorkflowDescriptorBuilderForSpecs {
 
   override implicit val actorSystem = system
-  implicit val DefaultDuration = 20.seconds.dilated
+  implicit val DefaultDuration = 60.seconds.dilated
```

Comment:
Why this change?

Reply:
This spec, like others, effectively runs its own entire Cromwell instance. When run on Travis it sometimes takes longer than 20 seconds for the lazy-loaded in-memory DB to start up, especially when other tests don't clean 100% clean up leading to GC pauses etc.

See the comment "Yes, this might be slow..." further down below in this file for more slightly more context.